openapi.yaml

```diff
@@ -6726,6 +6726,48 @@ paths:
       - lang: CLI
         source: >
           linode-cli lke pool-delete 12345 456
+  /lke/clusters/{clusterId}/pools/{poolId}/recycle:
+    parameters:
+    - name: clusterId
+      in: path
+      description: ID of the Kubernetes cluster to this Node Pool is attached to.
+      required: true
+      schema:
+        type: integer
+    - name: poolId
+      in: path
+      description: ID of the Kubernetes pool to be recycled.
```

Comment:
Can we change this to "ID of the Node Pool to be recycled."? since Node Pools are a provider construct and not actually a K8s concept.

Reply:
fixed